src/pages/Profile.jsx

```diff
@@ -89,24 +108,37 @@ const Profile = () => {
 
         <ProfileComponent clientProfile={clientProfile} />
 
-        <Button
-          variant="contained"
-          color="secondary"
-          size="small"
-          aria-label="Add Client Button"
-          startIcon={<AddIcon />}
-          onClick={() => setShowModal(true)}
-        >
-          Add Document
-        </Button>
-        <UploadDocumentModal showModal={showModal} setShowModal={setShowModal} />
-        <DocumentTable />
-        {!client && (
-          <>
-            <SetAclPermsDocContainerForm />
-            <SetAclPermissionForm />
-          </>
-        )}
+        <Container sx={{ display: 'flex', justifyContent: 'center', gap: 2 }}>
+          {!client && (
+            <Button
+              variant="contained"
+              color="primary"
+              size="small"
+              aria-label="Set Container Permissions Button"
+              startIcon={<SettingsIcon />}
+              onClick={() => handleAclPermissionsModal('container')}
+            >
+              Set Permission to All Documents
```

Comment:
I wonder if we could come up with more succinct phrasing for this text. But it could be difficult.

Reply:
"Share documents folder"

Reply:
Sounds good, I changed it to that